examples/experimental/tensorflow/mobilenet_v2/main.py

```diff
@@ -0,0 +1,159 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import subprocess
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import openvino.runtime as ov
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from tensorflow.python.keras.metrics import Metric
+
+import nncf
+from examples.experimental.tensorflow.mobilenet_v2.utils import center_crop
+
+# Path to the `mobilenet_v2` directory.
+ROOT = Path(__file__).parent.resolve()
+# Path to the directory where the original and quantized models will be saved.
+MODEL_DIR = ROOT / 'mobilenet_v2_quantization'
+# Path to ImageNet validation dataset.
+DATASET_DIR = Path("/ssd/tensorflow/imagenet") #ROOT / 'imagenet'
+
+
+def run_example():
+    """
+    Runs the MobileNetV2 quantization example.
+    """
+    # Step 1: Instantiates the MobileNetV2 from the Keras Applications.
+    model = tf.keras.applications.MobileNetV2()
+
+    # Step 2: Create calibration dataset.
+    data_source = create_data_source(batch_size=128)
```

Comment:
Why do we need batch size here? It can be hardcoded inside the function to make the samples consistent.

Reply:
Do you really think that we should align here?